.github/workflows/on_pull_request.yml

```diff
@@ -22,6 +22,31 @@ jobs:
       PR_NUMBER: ${{ github.event.pull_request.number }}
     secrets: inherit
 
+
+  cpp-paths-filter:
+    runs-on: ubuntu-latest
+    outputs:
+      cpp_changes: ${{ steps.filter.outputs.rerun_cpp }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          rerun_cpp:
+            - 'rerun_cpp/**'
```

Comment:
This misses a lot of stuff. Can we do `**/*.hpp`, `**/.cpp`, and `**/CMakeLists.txt` instead?

Reply:
I have no idea if this works anyways at this point